build-logic/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
     id("gradlebuild.collect-failed-tasks")
     id("gradlebuild.cache-miss-monitor")
+    id("gradlebuild.test-retry")
```

Comment:
❌ Like said above, don't apply this plugin in the root build, apply the plugin from the convention plugin.